llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -1084,6 +1234,12 @@ Error MCCASBuilder::createLineSection() {
 
   startSection(DwarfSections.Line);
 
+  // Create a block to the relocation addend values and zero the addends out in
+  // the section contents to improve deduplication.
+  if (auto E = createRelocAddendRefBlock(*DebugLineData)) {
+    return E;
+  }
```

Comment:
we can remove[ braces around single line if-statements](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements)